.gitignore

```diff
@@ -121,4 +121,4 @@ dist
 .DS_Store
 
 # yarn.lock
-yarn.lock
```

Comment:
What's this about?

Reply:
We put yarn.lock in the git ignore at some point (this was a few tickets ago, I don't remember which one off the top of my head...) because we were getting conflicts on merge repeatedly.

Reply:
We shouldn't be getting conflicts at all?

Reply:
I _think_ we commented this out of gitignore so that the virtual builds would have the right versions of dependencies, remembering now that we are running through these again.

Reply:
I still think we should remove the yarn lock here again, but I'm not gonna fight it.

Reply:
Pushed a brief clean-up of this branch and put yarn.lock back in gitignore; we can revisit later if we still have issues, but this seemed safer!